swiftsimio/metadata/cosmology/cosmology_fields.py

```diff
@@ -6,6 +6,8 @@
 
 # DEPRECATED: This information is now directly read out of the snapshots.
 
+# ALEXEI: mark everything as deprecated in addition to adding some bare-bones docs
```

Comment:
No need to docuemnt this code as it is completely unused. Can just remove these.